src/flux/course.coffee

```diff
@@ -0,0 +1,35 @@
+_ = require 'underscore'
+
+{TaskActions, TaskStore} = require './task'
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+
+CourseConfig =
+
+  loadPractice: (courseId) ->
+    @_local[courseId] ?= {}
+    @_local[courseId]['practices'] ?= []
+
+  loadedPractice: (obj, courseId) ->
+    @_local[courseId]['practices'].push(obj)
```

Comment:
oh hmm, cool okay. on http://localhost:3001/api/courses/1/practice there's only one.  that means i can take out some extra logic